docs_src/getting-started/Ch-GettingStartedCDevelopers.md

```diff
@@ -18,9 +18,25 @@ guide](./Ch-GettingStartedDevelopers.md), to build EdgeX C services, you will ne
 -   libcbor
 
 You can install these on Ubuntu by running:
-```bash
-sudo apt install libcurl4-openssl-dev libmicrohttpd-dev libyaml-dev libcbor-dev
+``` bash
+sudo apt-get install libcurl4-openssl-dev libmicrohttpd-dev libyaml-dev libcbor-dev
 ```
 
+CMake is required to build the SDKs.  Version 3 or better is required.  You can install CMAKE on Ubuntu by running:
+``` bash
+sudo apt install cmake
+```
+
+Check that your C development environment includes the following:
+
+- a version of GCC supporting C11
+- CMake version 3 or greater
+- Development libraries and headers for:
+    - curl (version 7.32 or later)
+    - microhttpd (version 0.9)
+    - libyaml (version 0.1.6 or later)
+    - libcbor (version 0.5)
+    - libuuid (from util-linux v2.x)
+
 ## Next Steps
 To explore how to create and build EdgeX device services in C, head to the [Device Services, C SDK guide](./Ch-GettingStartedSDK-C).
```

Comment:
Not among the changes in this PR, but worth fixing the link target.

s/Ch-GettingStartedSDK-C)/Ch-GettingStartedSDK-C.md)/